content/home/examples/an-application.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: An Application
+title: Una aplicación
 order: 2
 domid: todos-example
 ---
 
-Using `props` and `state`, we can put together a small Todo application. This example uses `state` to track the current list of items as well as the text that the user has entered. Although event handlers appear to be rendered inline, they will be collected and implemented using event delegation.
+Usando `props` y `state`, podemos integrar todo en una pequeña aplicación de tareas pendientes. Este ejemplo usa `state` para tener un control de la lista actual de objetos así como el texto que el usuario ha ingresado. A pesar que los `event handlers` parecen ser renderizados inline, estos serán recolectados e implementados usando delegación de eventos.
```

Comment:
“A pesar de que” instead of “a pesar que” [(see Fundéu)](https://www.fundeu.es/recomendacion/a-pesar-de-que-no-a-pesar-que-617/)
```suggestion
Usando `props` y `state`, podemos integrar todo en una pequeña aplicación de tareas pendientes. Este ejemplo usa `state` para tener un control de la lista actual de objetos así como el texto que el usuario ha ingresado. A pesar de que los manejadores de eventos parecen ser renderizados en línea, serán recolectados e implementados usando delegación de eventos.
```

Reply:
Sorry but  I think I will still use `event handlers` and `inline`, because I rarely hear people using its translations. But the `a pesar de que`, even if it sounds weird you're right that is the correct grammar